README.md

```diff
@@ -1,25 +1,30 @@
-# @api3/chains
-
-> The single source of truth for chain-related information
-
-- [Notes](#notes)
-  - [General](#general)
-  - [Deployment](#deployment)
-  - [hardhat-etherscan](#hardhat-etherscan)
-- [API](#api)
-  - [CHAINS](#chains)
-  - [hardhatConfig.networks()](#hardhatconfignetworks)
-  - [hardhatConfig.etherscan()](#hardhatconfigetherscan)
-  - [hardhatConfig.getEnvVariableNames()](#hardhatconfiggetenvvariablenames)
-  - [Types](#types)
-- [Scripts](#scripts)
-  - [generate:chains](#generatechains)
-  - [providers:ping](#providersping)
-- [Development](#development)
-  - [Validation](#validation)
-- [Building](#building)
-
-## Notes
+<h1 align="center">
+ @api3/chains ⛓️
+</h1>
+
+<div align="center">
+
+<a href="">![npm](https://img.shields.io/npm/v/%40api3%2Fchains)</a>
+<a href="">![npm](https://img.shields.io/npm/dw/%40api3/chains)</a>
+<a href="">![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/api3dao/chains/continuous-build.yml?label=Continuous Build)</a>
+<a href="">![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/api3dao/chains/check-providers.yml?label=Check Providers)</a>
+<a href="">![NPM](https://img.shields.io/npm/l/%40api3%2Fchains)</a>
+
+</div>
+
+<h4 align="center">
+The single source of truth for chain-related information across API3 projects
+</h4>
+
+## 📦 Installation
+
+```sh
+npm install @api3/chains --save
+yarn add @api3/chains
+pnpm add @api3/chains
+```
+
+## ❗ Important Notes
```

Comment:
Not sure if all of these notes are still relevant or not

Reply:
They are, I'm keeping them updated and find myself needing to refer to them regularly. However, you can move them to an issue/separate Markdown file, and they should be incorporated into the schema over time anyway (for example https://github.com/api3dao/chains/issues/91).

Reply:
While doing so, I would move the `hardhat-etherscan requires us to use a dummy API key with Blockscout block explorer APIs. We use "DUMMY_VALUE" but it could have been anything else.` note under https://github.com/api3dao/chains/blob/38ae666ecdb07b43ecbe1f4bdf6daf06f11764bb/README.md#hardhatconfigetherscan

Reply:
Extracted into `NOTES.md` - easier to maintain than an issue with a bunch of comments. Will open an issue to incorporate into the schema after merging

Reply:
https://github.com/api3dao/chains/issues/106